packages/gui/components/Select/Dropdown.tsx

```diff
@@ -64,48 +58,51 @@ const Dropdown: FC<Props> = ({ config }) => {
 					/>
 				</View>
 
-				{filterdItems.map((item, index) => {
-					const { id, name, icon } = getRequiredFields(item);
-					return (
-						<DropdownItem
-							key={index}
-							name={name}
-							icon={icon}
-							selected={!!selected && getRequiredFields(selected).id === id}
-							onPress={() => handleItemPress(item)}
-							style={styles.item}
-						/>
-					);
-				})}
-
-				{filterdItems.length == 0 && (
+				{filterdItems.length == 0 ? (
 					<View style={styles.notFoundView}>
 						<View style={styles.notFoundIconBlock}>
 							<SearchIcon size={17} color="#566674" />
 						</View>
 						<Text>{"We don't support this token yet!"}</Text>
 					</View>
+				) : (
+					<ScrollView
+						style={{
+							height: 200,
+						}}
```

Comment:
use style from `styles` (created by StyleSheet.create bellow), avoid inline style like this.